models/DbModel.js

```diff
@@ -1,5 +1,5 @@
 const DB_Define = require("../utils/DB_Define");
-const Define = require("../utils/Define");
+const Db_Define = require("../utils/Define");
```

Comment:
const Db_Define = require("../utils/Define");
replace this with this line
const Define = require("../utils/Define"); 

@Rijon1810